neet/boolean/logicnetwork.py

```diff
@@ -578,66 +578,38 @@ def _outgoing_neighbors_one_node(self, index):
 
         ::
 
-            >>> net = LogicNetwork([((1, 2), set(['11', '10'])), 
-                            ((0,), set(['1'])), 
-                            ((0, 1, 2), set(['010', '011', '101'])), 
+            >>> net = LogicNetwork([((1, 2), set(['11', '10'])),
+                            ((0,), set(['1'])),
+                            ((0, 1, 2), set(['010', '011', '101'])),
                             ((3,), set(['1']))])
-            >>> net._outgoing_neighbors_one_node(2)
+            >>> net.neighbors_out(2)
             set([0, 2])
         """
-        outgoing_neighbors = []
-        for i, incoming_neighbors in enumerate([list(row[0]) for row in self.table]):
+        outgoing_neighbors = set()
+        for i, incoming_neighbors in enumerate([row[0] for row in self.table]):
             if index in incoming_neighbors:
-                outgoing_neighbors.append(i)
+                outgoing_neighbors.add(i)
 
-        return set(outgoing_neighbors)
+        return outgoing_neighbors
 
-    def neighbors(self, index=None, direction='both'):
+    def neighbors(self, index):
         """
         Return a set of neighbors for a specified node, or a list of sets of
         neighbors for all nodes in the network.
 
         :param index: node index
         :param direction: type of node neighbors to return (can be 'in','out', or 'both')
```

Comment:
Need to update docstring since the in/out functions were promoted to public and removed from this function.